site.js

```diff
@@ -1187,13 +1257,25 @@ function generateCurse() {
 		},
 		{
 			makeSubjectText: function(){return isDecided(subjectFemale) ? subjectFemale ? "hen" : "rooster" : "chicken";},
-			chosen: function(){extemitiesName = "talons";},
+			chosen: function(){extemitiesName = "talons", facialfeatureName = randomFrom(["beak", "plumage"]);},
 			sets: [determinesRandomSex, mundaneAnimalSubject, setPussyName("cloaca"), setDickName("cloaca")],
 			requires: [becomingCreatureHybrid, beastOption],
 		},
+		{
+			subjectText: "dire rat",
+			chosen: function(){facialfeatureName = randomFrom(["whiskers", "pointed snout"]);},
+			sets: [determinesRandomSex, nonMundaneSubject],
+			requires: [beastOption],
+		},
 		{
 			subjectText: randomFrom(["hawk", "bluebird", "secretary bird"]), 
-			chosen: function(){extemitiesName = "talons";},
+			chosen: function(){extemitiesName = "talons", facialfeatureName = randomFrom(["beak", "plumage"]);},
+			requires: [genderAgnostic, beastOption],
+			sets: [mundaneAnimalSubject, setPussyName("cloaca"), setDickName("cloaca")],
+		},
+		{
+			makeSubjectText: function(){return isDecided(subjectFemale) ? subjectFemale ? "peahen" : "peacock" : "peafowl";},

```

Comment:
I want to combine this with the one above. Keep it a `makeSubjectText` because we need render-time logic (isSubjectFemale), but just make the peacock one another random option with the other birds.  Does that make sense? Just want to prevent a disproportionate amount of peacocks : P